packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -763,3 +764,70 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
     );
   }
 }
+
+/// A convenience class that wraps a [DropdownButton] in a [FormField] so as to
+/// act as a part of a [Form].
+///
+/// The [DropdownButton] widget is displayed without the underline, because it
+/// is provided by the [InputDecorator].
+///
+/// See also:
+///  * [DropdownButton], which is the underlying Widget providing the functionality
```

Comment:
nit: there needs to be a blank line before the line with the bullet point or it'll all get merged into one paragraph.